examples/next-forms/pages/js-form.js

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link'
+import styles from '../styles/Home.module.css'
+
+export default function PageWithJSbasedForm() {
+  // Handle the submit event on form submit.
+  const handleSubmit = async (event) => {
+    // Stop the form from submitting and refreshing the page.
+    event.preventDefault()
+
+    // Get data from the form.
+    const data = {
+      first: event.target.first.value,
+      last: event.target.last.value,
+    }
+
+    // Send the data to the server in JSON format.
+    const JSONdata = JSON.stringify(data)
+
+    // Send the form data to our forms API on Vercel and get a response.
```

Comment:
Could we remove Vercel from this? This would work self hosted, too 😄 
```suggestion
    // Send the form data to our API and get a response.
```